models/comments.model.js

```
@@ -10,3 +10,29 @@ exports.selectComments = ({ article_id }) => {
     return checkItemExists(rows) || rows;
   });
 };
+
+exports.insertComment = ({ username, body }, { article_id }) => {
+  if (username === undefined || body === undefined) {
+    return Promise.reject({ code: 400 });
+  }
+  const sql1 = "SELECT * FROM users WHERE username = $1";
```

If the username doesn't exist PSQL should throw an error for you because author is a foreign key and all foreign keys should be valid